src/app/shared/http-services/get-group-progress.service.ts

```diff
@@ -4,6 +4,14 @@ import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { appConfig } from 'src/app/shared/helpers/config';
 
+export interface Progress {
```

Comment:
All the progress fetching functions are contained in this single service (and file), that's why I put the 'Progress' interface in this file.

Reply:
Yest that's better. I thought this file only for "group progress" with its name. And I was probably expecting the "Progress" to be extended by several types in this case... but ok I understand. And is there a reason not to just use "TeamUserProgress" everywhere you use "Progress"?

Reply:
I guess not, I used the 'Progress' interface because I didn't use all the fields of the "TeamUserProgress" interface, so I removed it all together and use TeamUserProgress everywhere instead now.